dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/RepositoryUtility.java

```diff
@@ -283,7 +283,7 @@ public static RemoteRepository mavenRepositoryFromUrl(String mavenRepositoryUrl)
     }
 
     RemoteRepository repository =
-        new RemoteRepository.Builder(null, "default", mavenRepositoryUrl).build();
+        new RemoteRepository.Builder(mavenRepositoryUrl, "default", mavenRepositoryUrl).build();
```

Comment:
RemoteRepository's ID (the first argument) is nullable as per its Javadoc. However the Maven resolver requires different repositories to have different IDs.